### PR TITLE
Expand/Collapse Card Interaction

### PR DESCRIPTION
**Pull request by CurtisUnderwood**
### Additional Notes
- Added transition for expansion/collapse of universal benefit card
-
## Checklist

- [ ] Tests added for key features and bugs
- [x] Deployment working

### CODE REVIEW

**Comment by TishamIslam** [comment]
Not sure what happened, but I think it's having a weird interaction with the EI card since it has two sections of tasks.
image

After expansion it behaves normally.

**Review by shewood — CHANGES_REQUESTED** [review]
I see the same issue as Tisham but only on Firefox and only for that card.

**Comment by shewood** [comment]
I think the only change you need to make on the previous version is this to get it to do what we want.  
Change 
 block: 'start',
to
 block: 'nearest',

**Comment by CurtisUnderwood** [comment]
> I see the same issue as Tisham but only on Firefox and only for that card.

Good catch, seemed to be an issue only on Firefox can you confirm it's resolved?

> I think the only change you need to make on the previous version is this to get it to do what we want.

I think this may be a question for the PO, the card expansion ticket asks for a transition animation on the card dropdown which required a bit of a re-organization. Block 'nearest' would just remove the scroll, both are viable just depends if we want the animation.

File: components/molecules/UniversalBenefitCard.js [hunk 1]
```diff
@@ -97,7 +92,7 @@ export default function UniversalBenefitCard(props) {
       </h3>
       <div className=" flex flex-col bg-white  rounded-b-xl">
         {props.taskGroups == null || props.taskGroups.length <= 0 ? null : (
-          <div id={taskListId} className="  ">
+          <div className="  ">
```
Comment (shewood) [inline components/molecules/UniversalBenefitCard.js:95]: Looks good.
Any reason for not keeping the id on the tasklist?

**Pull request merged**